### PR TITLE
Update velero/operator-sdk/platformstatus

### PR DESCRIPTION
**Pull request by cblecker**
Sorry for the huge PR to those reviewing, but this does the following:
- Uses the github.com/cblecker/platformutils library to get infrastructure status
- Upgrades velero to 1.3.1
- Upgrades operator-sdk to 0.16.0

### CODE REVIEW

File: pkg/velero/crds.go [hunk 1]
```diff
@@ -17,9 +19,17 @@ func InstallVeleroCRDs(log logr.Logger, client client.Client) error {
 	var err error
 
 	// Install CRDs
-	for _, crd := range veleroInstall.CRDs() {
-		found := &apiextv1beta1.CustomResourceDefinition{}
-		if err = client.Get(context.TODO(), types.NamespacedName{Name: crd.ObjectMeta.Name}, found); err != nil {
+	for _, unstrcuturedCrd := range veleroInstall.AllCRDs().Items {
```
Comment (mbarnes) [inline pkg/velero/crds.go]: Nit: misspelling in `unstrcuturedCrd`

**Comment by cblecker** [comment]
/hold
for review comments

**Comment by cblecker** [comment]
Fixed spelling errors. Readding LGTM.

**Pull request merged**